app/uk/gov/hmrc/tariffclassificationfrontend/views/AssigneeCount.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.views
+
+import uk.gov.hmrc.tariffclassificationfrontend.models.{Case, Operator}
+
+case class AssigneeCount(op: Operator, count: Int) {
```

Comment:
I thought we'd have used a Mongo query to calculate the counts. Mongo's aggregation pipelines are built for this kind of thing I thought

Pol came up with this as an example `db.getCollection('cases').aggregate([{ $group: { _id: "$assignee" ,  count: { $sum: 1 } }}])`

Reply:
This approach was discussed and agreed with Ed.

Reply:
If we need it we can do the above later on, I suspect we wont though